README.md

```diff
@@ -83,8 +83,11 @@ some content
 ```
 
 Sometimes you don't want to include the layout in every file, so
-`gatsby-mdx` offers the option to set a default layout in the
-`gatsby-config.js` plugin config:
+`gatsby-mdx` offers the option to set default layouts in the
+`gatsby-config.js` plugin config. Set the key to the `name` set in the `gatsby-source-filesystem` config.
+If no matching default layout is found, the `default` default layout is used.
+
+You can also set `options.defaultLayout` if you only want to use one layout for all MDX pages.
```

Comment:
on a slight tangent: we should probably just make this a single option right? If it's a string then use it as the core default, if it's an object then check for the key.

Reply:
if we're going to mention it in the documentation that is. Otherwise, we can just deprecate the string option and remove it when we hit a 1.0